### ISSUE TITLE
Add a logger library to the project

### ISSUE THREAD

**Issue by vmarcosp** [iagolaguna/vscodefy-server#2]
I suggest add a logger library to the project. It's important for the server visibility.

**Comment by iagolaguna** [comment]
this server was deployed on Heroku and a have access to log in dashboard, but u can suggest some logger library to include in the project?

**Comment by vmarcosp** [comment]
Yes, of course. I like the (Winston)[https://github.com/winstonjs/winston] :smile:
